jenkins/pr-multi-build/Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+pipeline {
+    agent { label 'docker' }
+    parameters {
+        string(name: "NUM_RUNS", defaultValue: "3",
```

Comment:
note this really means 4 swatomation pipeline runs since 1 is done automatically through the existing check

Reply:
can we either update the comment to reflect the `-1` or do 

```groovy
def numRuns = Integer.parseInt("$NUM_RUNS") - 1
```

below and guard against negative numbers?

Reply:
I'd rather have the job run the number passed into it (imagining it could be very confusing to run it with `2` and have it only run once. The comment is mainly just about jenkins load, but I think it's fine since we're doing this on-demand instead of automatically with every PR.